tests/core_tests/startup_tests.py

```diff
@@ -0,0 +1,37 @@
+
+#!/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2020 - 2022 Pionix GmbH and Contributors to EVerest
+
+import logging
+import pytest
+
+from everest.testing.core_utils.fixtures import *
+from validations.base_functions import wait_for_and_validate_event
+from everest.testing.core_utils.test_control_module import TestControlModule
+from everest.testing.core_utils.everest_core import EverestCore
+from validations.user_functions import *
+
+@ pytest.mark.asyncio
+async def test_000_startup_check(everest_core: EverestCore):
+    logging.info(">>>>>>>>> test_000_startup_check <<<<<<<<<")
```

Comment:
doesn't pytest by itself report on which test it is running, or why do we need to log this ourself?